src/debugger/Debugger.js

```diff
@@ -59,14 +56,14 @@ export class Debugger {
       if (!this.proceedBreakpoint(lineNum, colNum)) return;
 
       console.log("Stopped for breakpoint on line " + lineNum);
-      this.sendDebugInfo(ast, contextStack, lineNum);
+      this.sendDebugInfo(ast, lineNum);
 
       let lastPoll = Date.now();
       let blocking = true;
       let contents = "";
       while (blocking) {
         if (Date.now() - lastPoll > 1000) {
-          contents = fs.readFileSync("./src/debugger/.sessionlogs/proxy2debugger.txt", "utf8").toString().split("\n");
+          contents = this.channel.readIn().toString().split("\n");
```

Comment:
Why the split? In general, should we be ignoring any characters?

Another issue: Once you've read a command from the file, shouldn't you make the file go away? If not, how do you know it has been updated?